script/merge_remote.py

```diff
@@ -49,7 +49,7 @@ def run_command(command, check=True, capture_output=True):
 def clone_repo(repo, temp_dir):
     """Clone the repository to a temporary directory using shallow clone."""
     print(f"Shallow cloning {repo}...")
-    # Use --depth=1 for shallow clone
+    # Keep the shallow clone for efficiency
```

Comment:
Is this what you meant?
```suggestion
    # Keep the clone shallow for efficiency
```